### PR TITLE
Add command for updating Module Twin

### PR DESCRIPTION
**Pull request by formulahendry**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ Interact with Azure IoT Hub, IoT Device Management, IoT Edge Management, IoT Hub
     * Get/update Device Twin
 * Interact with Azure IoT Edge <sup>Preview</sup> (Install [Azure IoT Edge](https://marketplace.visualstudio.com/items?itemName=vsciot-vscode.azure-iot-edge) for more IoT Edge support)
     * List Edge Moudles
-    * View Module Twin
+    * Get/update Module Twin
```
Comment (adashen) [inline README.md]: Edit?

File: src/constants.ts [hunk 2]
```diff
@@ -41,6 +44,10 @@ export class Constants {
     public static IoTHubAICreateDoneEvent = "AZ.IoTHub.Create.Done";
     public static IoTHubAIGetModuleTwinStartEvent = "AZ.Edge.ModuleTwin.Get.Start";
     public static IoTHubAIGetModuleTwinDoneEvent = "AZ.Edge.ModuleTwin.Get.Done";
+    public static IoTHubAIUpdateModuleTwinStartEvent = "AZ.Edge.ModuleTwin.Update.Start";
+    public static IoTHubAIUpdateModuleTwinDoneEvent = "AZ.Edge.ModuleTwin.Update.Done";
+    public static ModuleTwinJosnFileName = "azure-iot-module-twin.json";
+    public static ModuleTwinJosnFilePath = path.join(os.tmpdir(), Constants.ModuleTwinJosnFileName);
```
Comment (adashen) [inline src/constants.ts]: the module twin file is per workspace right? If so, shall we keep the file in this structure? I am considering the case when two vs code windows are open

File: README.md (hunk 1 above)
Comment (formulahendry) [reply README.md]: Updated, Thanks!

File: src/constants.ts (hunk 2 above)
Comment (formulahendry) [reply src/constants.ts]: Updated,

**Pull request merged**